developers/blobstreamx-offchain.md

```diff
@@ -1,18 +1,21 @@
 ---
-description: Learn how to integrate your L2's offchain logic with Blobstream
+description: Learn how to integrate your L2's offchain logic with Blobstream X
 ---
 
-# Integrate with Blobstream client
+# Integrate with Blobstream X client
 
-## Blobstream demo rollup
+## Blobstream X demo rollup
```

Comment:
```suggestion
## Blobstream demo rollup
```

Reply:
should these docs have the path `blobstreamx-offchain.md` or `blobstream-offchain.md`?

Reply:
that's a good question. I guess `blobstream-offchain.md` makes more sense since it's discussing how a Blobstream implementation would work, the general concept, not the zk implementation